ts/nni_manager/common/globals/unittest.ts

```diff
@@ -24,6 +24,7 @@ import path from 'path';
 
 import type { NniManagerArgs } from './arguments';
 import type { LogStream } from './log_stream';
+import { initLogStream } from './log_stream';
```

Comment:
For style consistency, please merge these lines to `import { LogStream, initLogStream } from ...`.

Reply:
updated